woocommerce/woocommerce-android
# [TTP] Add a button that leads to the support screen for the error that don't have resolution

Closes: #8301


### Description

The PR adds a "contact support" button to some not retriable error states

### Testing instructions

Modify `CardReaderPaymentViewState::332` - make it extend `ContactSupportError`
* Try to collect `1.03`
* Notice the "contact support" button
* Click on it leads to the contact support screen

### Images/gif


https://user-images.githubusercontent.com/4923871/234263603-eca71a0d-ebbf-4a62-9edb-916a2d037dad.mp4
